packages/number-field/test/inputs.test.ts

```diff
@@ -28,23 +27,6 @@ import {
 import { sendKeys } from '@web/test-runner-commands';
 
 describe('NumberField - inputs', () => {
-    before(async () => {
```

Comment:
CanIUse indicates that the international number field has been available in our supported evergreen - 2 so we can remove the polyfills now